src/core/errors.ts

```diff
@@ -4,6 +4,14 @@ import * as Ably from 'ably';
  * Error codes for the Chat SDK.
  */
 export enum ErrorCodes {
+  /**
+   * The message was rejected before publishing by a rule on the chat room.
+   *
+   * For example, a message may be rejected because it violates a moderation policy
+   * that bans profanity.
+   */
+  MessageRejectedByBeforePublishRule = 42210,
```

Comment:
I had to read this a few times because of the *By* in the middle, but I take it to mean it was rejected by a rule that runs before publish :D